en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-configuration.md

```diff
@@ -0,0 +1,22 @@
+# ISO8583 Connector Configuration
+
+ISO8583 is an international standard for financial transaction card originated interchange messaging. It is the International Organization for Standardization standard for systems that exchange electronic transactions initiated by cardholders using payment cards.
+
+Typically, whenever we use a credit card or debit card or an ATM card the data travels from one system to another system. A card based trantraction typically needs to travel between number of systems.
+ISO8583 is a financial transaction card originated in touch message message specifications adopted by a wide segment of payment industry which include acquiring and issuers.
```

Comment:
```suggestion
ISO8583 is a financial transaction card originated in touch message specifications adopted by a wide segment of the payment industry, which includes acquiring and issuers.
```

What do you mean by "acquiring" here?

Reply:
Changed as shown bellow.
"ISO 8593 is Financial Transaction card originated, Interchange message specifications adopted by wide segment of Payment Industry (including Acquirers and Issuers)."